docs/cmd/tkn_hub_search.md

```diff
@@ -0,0 +1,35 @@
+## tkn hub search
```

Comment:
Could we add some examples for `tkn hub search`? I guess this would actually need to happen in the hub cli, but it would be nice to have.